wpapi-class-content.php

```diff
@@ -7,18 +7,21 @@ function get_badresponse($wp_api_posts){
         $html .= "{$wp_api_posts['response']['message']}</dd></dl>";
         return $html;
     }
-    
+
     function get_posts($wp_api_posts){
         $html = '';
         foreach ($wp_api_posts as $k => $v){
-            $id = $v->ID;
-            $title = $v->title;
-            $link  = $v->link;
-            $thumbnail = $v->featured_image->guid;
-            $excerpt = $v->excerpt;
+            $id = $v['id'];
+            $title = $v['title']['rendered'];
+            $link  = $v['link'];
+            $excerpt = $v['excerpt']['rendered'];
             $html .= "<li><a href='{$link}'>";
-            if($thumbnail){$html .= "<img src='{$thumbnail}'>";}
-            $html .="<h2 class='wpapi-title'>{$title}</h2>{$excerpt}</a></li>";
+			if( isset( $v['_embedded']['https://api.w.org/featuredmedia'] ) && $v['_embedded']['https://api.w.org/featuredmedia'] ) {
+				$img_title = $v['_embedded']['https://api.w.org/featuredmedia'][0]['title']['rendered'];
+				$img_path   =$v['_embedded']['https://api.w.org/featuredmedia'][0]['media_details']['sizes']['thumbnail']['source_url'];

```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Line exceeds 120 characters; contains 136 characters